ynr/apps/people/models.py

```diff
@@ -245,6 +338,22 @@ def get_identifier(self, scheme):
             return identifier_object.identifier
         return ""
 
+    def get_identifiers_of_type(self, value_type=None):
+        qs = self.tmp_person_identifiers.all()
+        if value_type:
+            qs = qs.filter(value_type=value_type)
+        return qs
+
+    def get_single_identifier_of_type(self, value_type=None):
+        qs = self.get_identifiers_of_type(value_type=value_type)
+        if qs.exists():
+            return qs.first().value
```

Comment:
Should this raise if `len(qs)>1`? (at least for now)

Reply:
Happy to add that, but 66edf4646459157e32226ab40410d09194048d4d is setting this constraint at the DB level, so you could make the case for only using `get` here.

What do you reckon?

Reply:
oh yea - if only django provided some sort of built-in way to `filter()` and then raise if >1 objects are returned :D